Header header
string item
uint8 location
